l10ninsp/slave.py

```diff
@@ -123,6 +132,21 @@ def finished(self, *args):
       rc = FAILURE
     self.sendStatus({'rc': rc})
 
+def compareDirs(path1, path2):
+    curPath = os.path.dirname(os.path.abspath(__file__))
+    vendorLocalPath = os.path.join(curPath, '..', '..')
+    cljsPath = os.path.join(vendorLocalPath, 'compare-locales.js')
+    # nodejs ./bin/compare-dirs.js --data=json ~/elmo/stage/workdir/a
+    process = subprocess.Popen(
+      ["nodejs",
```

Comment:
we can always link node to nodejs. I'd prefer not to link to /usr/bin because it can be in a different dir. As long as it's in $PATH it should work.